src/main/java/teammates/common/datatransfer/attributes/EntityAttributes.java

```diff
@@ -60,15 +59,11 @@ public boolean isValid() {
     public abstract void sanitizeForSaving();
 
     /**
-     * Retrieves a list with nonempty errors from a list of error messages.
+     * Adds non-empty error message to a List, ignores empty error messages.
```

Comment:
Add @param tags to this javadoc? Also remember to mention that the errors parameter gets modified.